source/Mlos.Python/mlos/Optimizers/long_haul_tests/LongHaulTestBayesianOptimizer.py

```diff
@@ -167,9 +166,9 @@ def test_bayesian_optimizer_on_simple_2d_quadratic_function_cold_start(self):
 
             bayesian_optimizer.register(input_values_df, target_values_df)
             if i > 20 and i % 20 == 0:
-                self.logger.info(f"[{i}/{num_guided_samples}] Optimum: {bayesian_optimizer.optimum()}")
+                self.logger.info(f"[{i}/{num_guided_samples}] Optimum: {bayesian_optimizer.optimum()[1]}")
 
-        self.logger.info(f"Optimum: {bayesian_optimizer.optimum()}")
+        self.logger.info(f"Optimum: {bayesian_optimizer.optimum()[1]}")

```

Comment:
`[1]` seems like the more common value to be asking about, maybe it should be returned first?
Also, named values/properties seems maybe better.

Reply:
having configuration, optimum makes more sense to me, and I think actually the configuration is the more interesting one, but it's not used in the tests right now because it might not have been available before?

Reply:
We can always log both :)

Reply:
Also, are there any invariants we can assert about the optima in this test and in the tests below? Specifically:
1. That it is in fact the optimum
2. That the config belongs to the parameter space
3. That the optimum belongs to the objective space
4. Anything else you can think of

Reply:
I'm happy to add some of these but I feel that's mostly unrelated to this PR, right? More testing is always good but I'd love to push this to the students.